### PR TITLE
Allow write to file

### PR DESCRIPTION
**Pull request by phracek**
This allows container-workflow-tool to log some actions into a file specified by
--output-file option.
This option can help with automation.

Signed-off-by: Petr "Stone" Hracek phracek@redhat.com

### CODE REVIEW

File: container_workflow_tool/utility.py [hunk 1]
```diff
@@ -19,6 +19,8 @@ def print_help(self, file=None):
 
 # Utility textwrap functions
 def _2sp(a): return textwrap.indent(a, '  ')
+
+
```
Comment (pkubatrh) [inline container_workflow_tool/utility.py:23]: I guess this is forced by whatever style checker we are using right

File: container_workflow_tool/utility.py [hunk 2]
```diff
@@ -62,6 +64,10 @@ def _get_hostname_url(config):
     return git_url
 
 
+def save_output(logger, output):
```
Comment (pkubatrh) [inline container_workflow_tool/utility.py:67]: Not sure why we need this helper. Could we not just use logger.info?

File: container_workflow_tool/utility.py (hunk 1 above)
Comment (phracek) [reply container_workflow_tool/utility.py:23]: Yes, this is caught by style checker.

File: container_workflow_tool/utility.py (hunk 2 above)
Comment (phracek) [reply container_workflow_tool/utility.py:67]: I will try it and test it by next rebuild iteration.

**Comment by pkubatrh** [comment]
LGTM, thanks!

**Pull request merged**